sympy/physics/control/lti.py

```diff
@@ -1100,7 +1100,103 @@ def is_stable(self):
         True
 
         """
-        return fuzzy_and(pole.as_real_imag()[0].is_negative for pole in self.poles())
+        standard_form = self.to_standard_form()
+        return fuzzy_and(pole.as_real_imag()[0].is_negative for pole in standard_form.poles())
+
+    def to_standard_form(self):
+        r"""
+        Simplify the transfer function to its standard form.
+
+        Standard form : $\frac{a_n s^n + a_{n-1} s^{n-1} + \cdots + a_1 s + a_0}{b_m s^m + b_{m-1} s^{m-1} + \cdots + b_1 s + b_0}$
+
+        """
+        tf = self.doit().expand()
+        num, den = cancel(tf.num / tf.den).as_numer_denom()
+        return TransferFunction(num.collect(self.var), den.collect(self.var), self.var)
+
+    def get_asymptotic_stability_conditions(self):
+        """
+        Returns the asymptotic stability conditions for the transfer function.
+
+        Note: The set generated by the inequalities may be a subset of the real asymptotic stability conditions set
+        due to possible pole-zero cancellations.
+
+        Examples
+        ========
+
+        >>> from sympy import symbols, solve, reduce_inequalities
+        >>> from sympy.abc import s
+        >>> from sympy.physics.control.lti import TransferFunction, Feedback
+
+        >>> b1, b2, b3, b4 = symbols('b_1 b_2 b_3 b_4')
+        >>> p1 = b1*s**3 + b2*s**2 + b3*s + b4
+        >>> tf1 = TransferFunction(1, p1, s)
+        >>> tf1.get_asymptotic_stability_conditions()
+        [b_1 > 0, b_2 > 0, -b_1*b_4/b_2 + b_3 > 0, b_4 > 0]
+
+        >>> p2 = s**4 + 3*s**3 + 6*s**2 + 12*s + 8
+        >>> solve(p2)
+        [-2, -1, -2*I, 2*I]
+        >>> tf2 = TransferFunction(1, p2, s)
+        >>> tf2.get_asymptotic_stability_conditions()
+        False
```

Comment:
The method's name implies you will get conditions, so I find it an unusual result to get a boolean.